packages/grafana-ui/src/components/ValueMappingsEditor/MappingRow.tsx

```diff
@@ -7,36 +7,36 @@ import { MappingType, RangeMap, SelectableValue, ValueMap, ValueMapping } from '
 
 export interface Props {
   valueMapping: ValueMapping;
```

Comment:
Maybe change this for consistency?
```suggestion
  value: ValueMapping;
```

Reply:
this component has a lot of callbacks with a value argument:

```ts
export const MappingRow: React.FC<Props> = ({ valueMapping, onUpdate, onRemove }) => {
  const { type } = valueMapping;

  const onMappingValueChange = (value: string) => {
    onUpdate({ ...valueMapping, value: value });
  };

  const onMappingFromChange = (value: string) => {
    onUpdate({ ...valueMapping, from: value });
  };

  const onMappingToChange = (value: string) => {
    onUpdate({ ...valueMapping, to: value });
  };

  const onMappingTextChange = (value: string) => {
    onUpdate({ ...valueMapping, text: value });
  };
```

So change the prop to value would have some ripple effects